bazel_integration_test/private/default_test_runner.sh

```diff
@@ -42,6 +42,10 @@ done
 [[ -n "${workspace_dir:-}" ]] || exit_with_msg "Must specify the path of the workspace directory."
 [[ ${#bazel_cmds[@]} > 0 ]] || exit_with_msg "No Bazel commands were specified."
 
+for var_name in $ENV_VARS_TO_ABSOLUTIFY; do
+  export "${var_name}=$(pwd)/$(eval echo "\$${var_name}")"
```

Comment:
I don't understand why we need to do the `eval`. Is there a nuance with the evaluation of the `rootpath` that requires the `eval`?

Reply:
AFAICT in bash there's no way to print out an environment variable whose name is stored in an environment variable other than via `eval`...

We could alternatively generate a shell script which knows the keys, rather than sniffing them from an env var, or rewrite the bash into a non-bash language that actually lets you iterate over and set environment variables programatically, if that would be preferred?

Reply:
Would `printenv "${var_name}"` work?